clusterloader2/testing/load/modules/reconcile-objects.yaml

```diff
@@ -0,0 +1,268 @@
+## Input params
+# Valid actions: "creating", "scaling and updating", "deleting"
+{{$actionName := printf "%s objects" .actionName}}
+{{$namespaces := .namespaces}}
+
+# Derivative variables
+{{$is_creating := (eq .actionName "creating")}}
+{{$is_scaling := (eq .actionName "scaling and updating")}}
+{{$is_deleting := (eq .actionName "deleting")}}
+{{$tuning_set := "N/A"}}
+{{if $is_creating}}{{$tuning_set = "RandomizedSaturationTimeLimited"}}{{end}}
+{{if $is_scaling}}{{$tuning_set = "RandomizedScalingTimeLimited"}}{{end}}
+{{if $is_deleting}}{{$tuning_set = "RandomizedDeletionTimeLimited"}}{{end}}
+{{$randomScaleFactor := DefaultParam .randomScaleFactor 0}}
+{{$minReplicaFactor := SubtractFloat 1 $randomScaleFactor}}
+{{$maxReplicaFactor := AddFloat 1 $randomScaleFactor}}
+
+# DaemonSets
+{{$daemonSetImage := DefaultParam .daemonSetImage "k8s.gcr.io/pause:3.0"}}
```

Comment:
I don't think we change deamonset image anywhere so I guess we could remove this parameter for now.

Reply:
Done.

Reply:
Actually I'm reverting this to the old state. The `daemonSetImage` is actually changed between module definitions.

Reply:
Ah, you are right, haven't noticed that ;)